shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -65,6 +66,11 @@ @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegat
  */
 @property(nonatomic, assign) double targetViewInsetBottom;
 @property(nonatomic, retain) VSyncClient* keyboardAnimationVSyncClient;
+@property(nonatomic, assign) Boolean keyboardAnimationIsShowing;
+@property(nonatomic, assign) Boolean keyboardAnimationIsCompounding;
+@property(nonatomic, assign) fml::TimePoint keyboardAnimationStartTime;
+@property(nonatomic, assign) CGFloat keyboardAnimationFrom;
+@property(nonatomic, assign) CGFloat keyboardAnimationTo;
```

Comment:
1. I think keyboardAnimationTo can be deleted, `targetViewInsetBottom` is for the animation target value. And there is no need to create a copy of it O(∩_∩)O
2. Maybe `keyboardAnimationFrom` can be renamed with `originalViewInsetBottom`, corresponding to `targetViewInsetBottom`